### ISSUE TITLE
docker-compose issue

### ISSUE THREAD

**Issue by robelk123** [kebronDaniel/buchi#1]
when i called docker-compose up -d

ERROR: Service 'api' depends on service 'mongo_db' which is undefined.

**Comment by kebronDaniel** [comment]
You are right it was a typo error, I defined the db as 'mongo-db' but used mongo_db in the connection string.

**Comment by kebronDaniel** [comment]
You are right, It was a typo error.
I have made a new commit.
Sorry about that, I was so consumed in the test code and the image upload
functionalities that I didn't completely test the dockerized version.

Thank you!

**Comment by robelk123** [comment]
it's ok. let me check
________________________________

**Comment by robelk123** [comment]
hey kebron

i'm facing another error.

please test everything well before submitting. let me give you an hour.


________________________________
